AppInspector.RulesEngine/OatExtensions/OatRegexWithIndexOperation.cs

```diff
@@ -86,6 +86,14 @@ private static IEnumerable<Violation> RegexWithIndexValidationDelegate(CST.OAT.R
             RegexOptions regexOpts = Common.Utils.RegexModifierToRegexOptions(src.Arguments);
             var subBoundary = state2 is Boundary s2 ? s2 : null;
 
+#if NET7_0_OR_GREATER
```

Comment:
Im not sure this is the right place to add this. The validation delegate isn't guaranteed to run. Should likely be in the RegexModiferToRegexOptipns method instead.

Reply:
Probably, this is a new method to me, I was asking the same question myself.

Reply:
Yes, I just added this yesterday so understandable you may have missed it in the merge. I was hoping to make it a little easier to expand the modifiers available with the new method.